shell/platform/android/test/io/flutter/plugin/platform/PlatformViewsControllerTest.java

```diff
@@ -361,31 +361,32 @@ public void itUsesActionEventTypeFromFrameworkEventAsActionChanged() {
     assertNotEquals(resolvedEvent.getAction(), frameWorkTouch.action);
   }
 
-  @Ignore
   @Test
-  public void itUsesActionEventTypeFromMotionEventForHybridPlatformViews() {
+  public void toMotionEvent_returnsSameCoordsForVdAndNonVd() {
     MotionEventTracker motionEventTracker = MotionEventTracker.getInstance();
     PlatformViewsController platformViewsController = new PlatformViewsController();
 
     MotionEvent original =
         MotionEvent.obtain(
-            100, // downTime
-            100, // eventTime
-            1, // action
-            0, // x
-            0, // y
+            10, // downTime
+            10, // eventTime
+            261, // action
+            1, // x
+            1, // y
             0 // metaState
             );
 
-    // track an event that will later get passed to us from framework
+    // Get the result of toMotionEvent for both the virtual display and non virtual display case,
+    // and make sure they are identical in their x and y coordinates.
+
     MotionEventTracker.MotionEventId motionEventId = motionEventTracker.track(original);
 
-    PlatformViewTouch frameWorkTouch =
+    PlatformViewTouch frameWorkTouchNonVd =
         new PlatformViewTouch(
             0, // viewId
             original.getDownTime(),
             original.getEventTime(),
-            2, // action
+            0, // action
```

Comment:
why does the action change? can we just use original.getAction() ?

Reply:
Yes, I copied this from an existing test but this makes much more sense to me